testcases/OpalErrorLog.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+# IBM_PROLOG_BEGIN_TAG
+# This is an automatically generated prolog.
+#
+# $Source: op-test-framework/testcases/OpalErrorLog.py $
+#
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2015
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+# IBM_PROLOG_END_TAG
+
+#  @package OpalErrorLog
+#  Currently runs only in FSP platforms
+#
+
+import time
+import subprocess
+import re
+
+from common.OpTestIPMI import OpTestIPMI
+from common.OpTestConstants import OpTestConstants as BMC_CONST
+from common.OpTestError import OpTestError
+
+import unittest
+import OpTestConfiguration
+from common.OpTestSystem import OpSystemState
+
+class OpalErrorLog(unittest.TestCase):
+    def setUp(self):
+        conf = OpTestConfiguration.conf
+        self.cv_SYSTEM = conf.system()
+        self.cv_FSP = self.cv_SYSTEM.bmc
+        self.cv_HOST = conf.host()
+        self.bmc_type = conf.args.bmc_type
+        self.cv_SYSTEM.goto_state(OpSystemState.OS)
+
+    def tearDown(self):
+        self.cv_HOST.host_gather_opal_msg_log()
+        self.cv_HOST.host_gather_kernel_log()
+
+
+    ##
+    # @brief This function tests OpalErrorLog component
+    #
+    # @return BMC_CONST.FW_SUCCESS or raise OpTestError
+    #
+    def runTest(self):
+        if "FSP" not in self.bmc_type:
```

Comment:
yeah, It's currently only useful for FSP platforms, I don't the method till yet how to generate an error log from OPAL for testing.(We know other methods in real time -Injecting processor errors or disable some devices like occ, But which is will impact the system needs the IPL's).